### ISSUE TITLE
Planetwars rank should not have any effect on anything

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K-Infrastructure#2217]
Currently PW rank is giving some people higher star colour rank than they would otherwise have. This is an effect of PW rank that should be removed. I don't know of any others but they should be removed too.

**Comment by Licho1** [comment]
This sounds like a bug .. @DeinFreund  ?

**Comment by DeinFreund** [comment]
I would suggest a pw rank reset for now. 

If you think it's a bug I can remove it before starting PW.

**Comment by DeinFreund** [comment]
The nice part about this is that it gives the good PW players a reward. The percentile requirements are the same as in the matchmaking and casual category. 

The only reason it's considered easier to gain ranks via PW is that some of the top players weren't playing in it, allowing others to get to those ranks. There's already a requirement of at least 50 active PW players before anyone can gain ranks from it, this number can be changed arbitrarily. 

I've chosen 50 because with ~60 active players, matchmaking is not much more representative. The difference only lies in some players from mm not playing PW.

**Comment by sprunk** [comment]
>Is there something speaking against rewarding top pw players just like we do top mm and casual players?

Yes, we already give them planets.

**Comment by DeinFreund** [comment]
No, I was #1 at a planetwars without receiving a single planet. Planet distribution is special.

**Comment by sprunk** [comment]
So improve the planet distribution system then.
